src/components/EmailScreen.tsx

```diff
@@ -1,51 +1,91 @@
-import { BodyText, CaptionText } from 'components/Text'
+import { CaptionText } from 'components/Text'
+import AnonFace from 'icons/AnonFace'
 import Button from 'components/Button'
 import EmailScreenParams from 'models/EmailScreenParams'
 import classnames, {
   alignItems,
+  blur,
   display,
   flex,
   flexDirection,
   gap,
+  inset,
   justifyContent,
+  maxHeight,
+  minHeight,
+  minWidth,
+  objectFit,
+  overflow,
+  padding,
+  position,
+  zIndex,
 } from 'classnames/tailwind'
 import isMobileDevice from 'helpers/isMobileDevice'
 import openAppEmailLink from 'helpers/openAppEmailLink'
 
+const videoContainer = classnames(
+  position('absolute'),
+  inset('inset-0'),
+  blur('blur-xl'),
+  minWidth('min-w-full'),
+  minHeight('min-h-full'),
+  maxHeight('max-h-full'),
```

Comment:
Why do we need min-max with same valuees? Simply use `height(h-full)`

Reply:
Changed it